controllers/jobs.js

```diff
@@ -0,0 +1,88 @@
+
+
+const Job = require('../models/Job');
+const { StatusCodes } = require('http-status-codes')
+const { BadRequestError, NotFoundError } = require('../errors')
+
+// Create a Job
+// POST api/job
+const createJob  = async (req, res) => {
+    req.body.createdBy = req.user.userId
+    const job = await Job.create(req.body);
```

Comment:
**Suggestion:** Instead of changing the request body directly and passing it for object creation, you might want to create a separate object. That way you have control over everything that is passed for your database, even if the user passes extra properties, you won't use it.


```suggestion
    const { company, position, status } = req.body;
    req.body.createdBy = req.user.userId;
    const job = await Job.create({
        company,
        position,
        status,
        createdBy: req.user.userId,
    });
```